runtimes/polimec/src/xcm_config.rs

```diff
@@ -417,11 +417,10 @@ impl pallet_xcm::Config for Runtime {
 	// ^ Disable dispatchable execute on the XCM pallet.
 	// Needs to be `Everything` for local testing.
 	type XcmExecutor = XcmExecutor<XcmConfig>;
-	// We only allow reserve based transfers of AssetHub reserve assets back to AssetHub.
-	type XcmReserveTransferFilter = AssetHubAssetsAsReserve;
+	// We allow all reserve based transfers from our chain to any other chain.
+	type XcmReserveTransferFilter = Everything;
```

Comment:
It would not mean that we would accept any reserve transfer, as those are filtered out by the executor (see IsReserve). We still only accept USDT, USDC and DOT from AssetHub as reserve transfers. 
It does mean that reserve transfers to places that are not supported is possible, however from my understanding of the XcmReserveTransferFilter, there is no "destination" check. The check is only for the origin executing the XCM and the Asset. So we can filter out certain assets from Reserve based transfers, but we cannot filter out destination chains. As we only have currently 4 assets (USD(T/C), DOT and PLMC) and we allow reserve based transfers of all of them, we can allow Everything.